angular/src/app/utils/questionnaireBuilder.ts

```diff
@@ -0,0 +1,2990 @@
+import * as $ from 'jquery';

```

Comment:
Maybe include top-level info on this file.  where code is from and what we are trying to do here.  so some history/tips/overview for a future developer who is looking at this part.  maybe some notes (like to good drive or attachments from google drive) could be put a jira issue that we refer to here.  🤔 .  somehow need to credit the author/source 🤔 

same for angular/src/app/components/modal-questionnaire-viewer/modal-questionnaire-viewer.component.styl